frontend/src/app/leaderboard/table.tsx

```
@@ -0,0 +1,147 @@
+  return (
+    <div>
+      <div className="overflow-x-auto mt-10">
+        <table className="table">
+          {/* head */}
+          <thead>
+            <tr>
+            <th>Rank</th>
+              <th>Name</th>
+              <th>Job</th>
+              <th>Favorite Color</th>
+              <th></th>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)